Library/CurrencyPercentageTier.cs

```diff
@@ -0,0 +1,71 @@
+using System;
+using System.Collections.Generic;
+using System.Xml;
+
+namespace Recurly 
+{
+  /// <summary>
+  /// Represents add-on tiers
+  /// </summary>
+  public class CurrencyPercentageTier : RecurlyEntity
+  {
+    public string Currency { get; set; }
+
+    private List<PercentageTier> _percentageTiers;
+
+    public List<PercentageTier> PercentageTiers
+    {
+        get { return _percentageTiers ?? (_percentageTiers = new List<PercentageTier>()); }
+        set { _percentageTiers = value; }
+    }
+
+    #region Constructors
+
+    public CurrencyPercentageTier() {
+    }
+
+    internal CurrencyPercentageTier(XmlTextReader xmlReader){
+      ReadXml(xmlReader);
```

Comment:
ifd do this but up to you

`internal CurrencyPercentageTier(XmlTextReader xmlReader)
{
      ReadXml(xmlReader);
 }`

Reply:
yikes its hard to get styling right in these comments